src/io_array.ml

```diff
@@ -78,7 +78,7 @@ module Make (IO : Io.S) (Elt : ELT) :
       Log.warn (fun m ->
           m "Requested pre-fetch region is empty: [%Ld, %Ld]" low high)
     else if range > max_buffer_size then
-      Log.debug (fun m ->
+      Log.warn (fun m ->
```

Comment:
Why is this a `warn`? Doesn't this happen frequently since `interpolation_search` optimistically pre-fetches?

Reply:
I think the fanout is supposed to create < 4096 bytes buckets, so the range of search of the interpolation search should always be smaller.